content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -71,6 +71,12 @@ An _output entry_ (`outs`) consists of these fields:
   _checksum_ for HDFS.
 - `cache`: Whether or not this file or directory is <abbr>cached</abbr> (`true`
   by default, if not present). See the `--no-commit` option of `dvc add`.
+- `store`: Whether this file or directory can be
+  [pushed](/doc/command-reference/push) and
+  [pulled](/doc/command-reference/pull) to/from
+  [remote storage](/doc/command-reference/remote) (`true` by default for
+  all outputs except for imports, where it is `false` by default). See
```

Comment:
> except for imports

Per https://github.com/iterative/dvc.org/issues/1865#issuecomment-714433118 this isn't the only exception. External outputs are also never stored remotely (since they are already in an external cache).